### ISSUE TITLE
[BUG] Ajax tests fail

### ISSUE THREAD

**Issue by dingo-d** [lucatume/wp-browser#655]
**Environment**
OS: MacOS Ventura
PHP version: 8.1.23
Installed Codeception version: 5.0.11  
Installed wp-browser version: 4.0.3  
WordPress version: 6.3  
Local development environment: Valet
WordPress structure and management: Default

**Can you perform the test manually?**
Yes, the code works when I manually trigger the AJAX callback.

**Codeception configuration file**  

```yaml
namespace: Tests
support_namespace: Support
paths:
    tests: tests
    output: tests/_output
    data: tests/Support/Data
    support: tests/Support
    envs: tests/_envs
actor_suffix: Tester
params:
    - tests/.env
coverage:
    enabled: true
    low_limit: 30
    high_limit: 80
    include:
        - src/*
extensions:
    enabled:
        - Codeception\Extension\RunFailed
        - lucatume\WPBrowser\Extension\ChromeDriverController
        - lucatume\WPBrowser\Extension\BuiltInServerController
    config:
        lucatume\WPBrowser\Extension\ChromeDriverController:
            port: '%CHROMEDRIVER_PORT%'
        lucatume\WPBrowser\Extension\BuiltInServerController:
            workers: 5
            port: '%BUILTIN_SERVER_PORT%'
            docroot: '%WORDPRESS_ROOT_DIR%'
            env:
                DATABASE_TYPE: sqlite
                DB_ENGINE: sqlite
                DB_DIR: '%codecept_root_dir%/tests/Support/Data'
                DB_FILE: db.sqlite
    commands:
        - lucatume\WPBrowser\Command\RunOriginal
        - lucatume\WPBrowser\Command\RunAll
        - lucatume\WPBrowser\Command\GenerateWPUnit
        - lucatume\WPBrowser\Command\DbExport
        - lucatume\WPBrowser\Command\DbImport
        - lucatume\WPBrowser\Command\DevStart
        - lucatume\WPBrowser\Command\DevStop
        - lucatume\WPBrowser\Command\DevInfo
        - lucatume\WPBrowser\Command\DevRestart
        - lucatume\WPBrowser\Command\ChromedriverUpdate
```

**Suite configuration file**
Paste, in a fenced YAML block, the content of the suite configuration file; remove any sensitive data!

```yaml
# Integration suite configuration
#
# Run integration and "WordPress unit" tests.

actor: IntegrationTester
bootstrap: _bootstrap.php
modules:
    enabled:
        - lucatume\WPBrowser\Module\WPLoader
    config:
        lucatume\WPBrowser\Module\WPLoader:
           wpRootFolder: "%WORDPRESS_ROOT_DIR%"
           dbUrl: '%WORDPRESS_DB_URL%'
           wpDebug: true
           tablePrefix: '%TEST_TABLE_PREFIX%'
           domain: '%WORDPRESS_DOMAIN%'
           adminEmail: 'admin@%WORDPRESS_DOMAIN%'
           title: 'Integration Tests'
           plugins: ['woocommerce/woocommerce.php','wp-orderprotection/wp-orderprotection.php']
           theme: 'storefront'
```

**Describe the bug**
I'm trying to test out the AJAX callback in my plugin, and my test looks like this:

```php
<?php

namespace Tests\Integration\Ajax;

use lucatume\WPBrowser\TestCase\WPAjaxTestCase;
use myAjaxClass;

class SomeClassTest extends WPAjaxTestCase
{
	public function testAjaxCallbackWillFailIfNumberNotOnlyOnceIsInvalid()
	{
		try {
			$this->_handleAjax(myAjaxClass::AJAX_CALLBACK_HANDLER);
		} catch (\WPAjaxDieStopException $e) {
			$response = json_decode($this->_last_response);
			
			$this->assertNotNull($response);
			$this->assertNotEmpty($response->error);
		}
	}
}
```

**Output**

```bash
> @php ./vendor/bin/codecept run Integration
Codeception PHP Testing Framework v5.0.11 https://helpukrainewin.org
ChromeDriver already running.
PHP built-in server already running.

Tests.Integration Tests (3) -----------------------------------------------------------------------------------------------------------------------------------------------------------
- SomeClassTest: Ajax callback will fail if number not only once is invalid{"success":false,"data":"Number not only once is invalid"}


COMMAND DID NOT FINISH PROPERLY.
Script @php ./vendor/bin/codecept run Integration handling the test:integration event returned with error code 125
```

So the actual output is shown, because my callback looks like this:

```php
public function getAjaxCallback(): void
{
	if (
		!isset($_POST[self::NONCE_NAME]) ||
		!wp_verify_nonce(sanitize_key($_POST[self::NONCE_NAME]), self::NONCE_ACTION)
	) {
		wp_send_json_error(esc_html__('Number not only once is invalid', 'my-plugin'), 404);
	}

	// The rest of the code...
}
```

So the correct error is shown (`wp_send_json_error` is just a fancy wrapper around `wp_die` anyhow).

**To Reproduce**
Try to create AJAX callback and write the test for it 🤷🏼‍♂️ 

**Expected behavior**
The error should be caught in the `$_last_response` property.

### Unrelated note to the bug

The docs website seems to be down.

**Comment by lucatume** [comment]
Thank you @dingo-d  for taking the time to open an issue and provide the information.
I could reproduce the issue: it's caused by a broken inheritance chain in the tests.
I'm working to fix this and will post updates in this issue when done.

**Comment by lucatume** [comment]
Fixed in 4.0.7, I've reproduced the example test provided [here](https://github.com/lucatume/wp-browser/blob/fa05d635f50a4ffef54e39b03aabed0ab42882bb/tests/wploadersuite/AjaxTest.php#L20-L19)
